case-lib/pipeline.sh

```diff
@@ -7,46 +7,39 @@ func_pipeline_export()
         dlogi "Topology file name is not specified, unable to run command: ${BASH_SOURCE[-1]}" && exit 1
     fi
 
-    local tplg="$1" opt="" sofcard=${SOFCARD:-0} cmd=""
+    local tplg="$1" ignore="" sofcard=${SOFCARD:-0}
```

Comment:
older design `opt` is base on when `$*` have some option like `-s` can also pass-down to here  
so it seems have some confuse  
you also use `opt` and `cmd`

Reply:
This function is used to export pipeline, the options we used here is -f -b and -s,  and -s sofcard is here : `sofcard=${SOFCARD:-0}`, so here only process filter string and block option is enough.
Has checked all other option of sof-tplgreader.py, no other option can be used here.

Reply:
`opt` and `cmd` also been used, so you can add them back